_pages/rentals/index.md

```diff
@@ -11,7 +11,7 @@
 
 <div class="alert alert-warning">
   In accordance with <a href="http://web.mit.edu/covid19/">MIT's response to COVID-19</a>,
-  <strong>all MITOC office hours are canceled indefinitely.</strong>
+  <strong>all MITOC office hours are canceled indefinitely.</strong> We are unable to access the office, and are thus <strong>unable to rent gear</strong> until further notice.
```

Comment:
Might consider breaking the "We are unable to access the office" onto a new line?
No `<p>` tags:
<img width="1170" alt="image" src="https://user-images.githubusercontent.com/901169/94067170-51f8a380-fda2-11ea-9757-a0a3b5add63c.png">

Two paragraphs:
<img width="1163" alt="image" src="https://user-images.githubusercontent.com/901169/94067237-6dfc4500-fda2-11ea-8c4b-06b564d1b314.png">